### PR TITLE
Promote LoadBalancerIPMode to Beta

### PR DESCRIPTION
**Pull request by rikatz**
As part of promotion of LoadBalancerIPMode to Beta, this is a placeholder PR to properly update the docs

### CODE REVIEW

**Comment by rikatz** [comment]
/hold

This should be merged just after the main code is merged, I will keep this updated here

**Review by dipesh-rawat — COMMENTED** [review]
@rikatz We should consider setting the `dev-1.30`  branch as the target branch rather than the `main` branch for this pull request. Since the change is related to the feature targeting beta in 1.30.

**Comment by rikatz** [comment]
Hum, did but will need to rebase tomorrow

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -672,7 +672,7 @@ Unprefixed names are reserved for end-users.
 
 {{< feature-state for_k8s_version="v1.29" state="alpha" >}}
```
[the remark this answers is not in the document]
Comment (rikatz) [reply content/en/docs/concepts/services-networking/service.md]: fixed

[the remark this answers is not in the document]
Comment (rikatz) [reply content/en/docs/concepts/services-networking/service.md]: actually apparently someone already fixed it before, as when I rebased it was conflicting

**Comment by rikatz** [comment]
/hold cancel
https://github.com/kubernetes/kubernetes/pull/123418 is the PR for the feature promotion on k/k

**Comment by rikatz** [comment]
/assign @dipesh-rawat 

This is ready to be merged, the PR on k/k is done. Please let me know if there's anything else to be done here :)

**Review by dipesh-rawat — COMMENTED** [review]
Thanks! Changes look good to me. It would be nice to update the PR [description](https://github.com/kubernetes/website/pull/45219#issue-2143212876) to reflect that it's no longer a placeholder. Reviewers/Approvers can now review it, and mentioning the upstream PR https://github.com/kubernetes/kubernetes/pull/123418 would also be helpful.

/lgtm

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -672,7 +672,7 @@ Unprefixed names are reserved for end-users.
 
 {{< feature-state feature_gate_name="LoadBalancerIPMode" >}}
 
-Starting as Alpha in Kubernetes 1.29, 
+Starting as Beta in Kubernetes 1.30,
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/service.md:675]:
```suggestion
As a Beta feature in Kubernetes 1.30, 
```

**Comment by rikatz** [comment]
@tengqm change applied as requested

**Comment by ramrodo** [comment]
Thanks!
/lgtm

**Pull request merged**